### PR TITLE
add txn lifecycle nb

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb [hunk 1]
```diff
@@ -0,0 +1,709 @@
+{
```
Comment (rizwansaeed) [inline examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb:1]: <p>There's an assumption here that the status can only be in one state, is that valid? In some conversations there has been a view that there can be multiple states and each on would be its own property. I think what we have here is still a great example and may actually be enough for some use cases, we should check to see if we need the other example too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (rizwansaeed) [inline examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb:1]: <p>could we rename the columns for clarity?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (slemasne) [reply examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb:1]: <p>Agreed. </p><p>In latest PR I have added two new properties to represent two new statuses:</p><p>confirmation_status</p><p>settlement_status</p>

Comment (slemasne) [reply examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb:1]: <p>Sure. Done in latest PR. </p>

File: examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb [hunk 2]
```diff
@@ -0,0 +1,806 @@
...
```
Comment (rizwansaeed) [inline examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb:1]: <p>would these be better as a dict?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (rizwansaeed) [inline examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb:1]: <p>can we update the (property) column names here as well?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (slemasne) [reply examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb:1]: <p>Sure. Updated. </p>

Comment (slemasne) [reply examples/use-cases/post-trade-processing/Managing a transaction lifecycle using LUSID's properties.ipynb:1]: <p>Agreed.</p><p>Updated. </p>

**Pull request merged**